en/developer_guide/other_database_type.rst

```diff
@@ -0,0 +1,54 @@
+Using other database type
+=========================
+
+Context
+-------
+
+Open orchestra uses by default a MongoDB database. If you want to use another type of database like
+SQL, you will need to tune some settings and to replace some files.
+
+Configuration: object manager
+-----------------------------
+
+By default, Open Orchestra uses Doctrine ODM to map database documents to objects. Developers can
+define another document manager, the only condition is to implement
```

Comment:
must not can
This sentence is weird : 
Developers must define another document manager which implement Doctrine\Common\Persistence\ObjectManager like EntityManager for instance in configuration